x-pack/solutions/security/plugins/security_solution/public/kubernetes/pages/index.tsx

```diff
@@ -8,6 +8,7 @@
 import React, { useCallback, useMemo } from 'react';
```

Comment:
@animehart as far as I understand, the `kuberneted` page is behind the feature flag. Is it already possible to test it, eg. with some mocked data? if not, I guess we can approve and test when we have the data from d4c